package.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "notify-watcher-telegram-bot",
```

We should have all packages as scope ones

```suggestion
  "name": "@notify-watcher/telegram-bot",
```